### PR TITLE
Fixes typos, moves link, removes images, adds Grammarly

### PR DESCRIPTION
**Pull request by Annamarie2019**
These changes must have been in the wrong project, so didn't make it in. None are substantive, but will make the article more professional looking.

- Fixed typos, like # instead of a $ sign, and missing periods and line spaces.
- Moved CID gravity link up to deals section, where it's more relevant (I promised Julien, the developer, I'd lpoint him to the doc when it was finished, so would be great if his link is in the ideal place.)
- Corrects image formatting, so images should show now....one image on costs and one on vibrant ecosystem (from Andrew Stanco's [Filecoin Economics](https://www.youtube.com/watch?v=MSK9S_J1w1Q&t=446s) )
- Adds Grammarly suggestions and other minor details

### CODE REVIEW

File: docs/storage-provider/how-providing-works.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+---
+title: 'How providing storage works'
+description: 'Providing storage to the Filecoin network involves accepting deals and honoring them by inserting proofs in the chain.'
+---
+
+# {{ $frontmatter.title }}
```
Comment (johnnymatthews) [inline docs/storage-provider/how-providing-works.md:6]:
```suggestion
```

We don't use `$frontmatter.title` anymore, since we moved over to the Hugo platform.

File: docs/storage-provider/how-providing-works.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+Here's a general idea of the minimum hardware you'll need for providing storage:
+
+- An 8+ core CPU
+- 128 GiB of RAM at the very least
+- A powerful GPU to speed up SNARK computations
+- 1TiB NVMe-based disk space for cache storage
+- Additional hard drives for storing _sealed sectors_, the Lotus chain, and more.
+
+For more detail, see:
+- [Hardware requirements](./storage-provider/hardware-requirements/#general-hardware-requirements)
```
Comment (johnnymatthews) [inline docs/storage-provider/how-providing-works.md:43]:
```suggestion
- [Hardware requirements]({{< relref "storage-provider/hardware-requirements/#general-hardware-requirements" >}})
```

File: content/en/storage-provider/how-providing-works/index.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+- Block reward as collateral (vesting block rewards over time)
+- Storage deal provider collateral (aligns incentives between storage provider and client)
+
+See [Filecoin Spec](https://spec.filecoin.io/#section-systems.filecoin_mining.miner_collaterals).
+
+## Gas fees
+
+You must pay a nominal gas fee for resources used for sealing storage. You can set a gas limit, above which a transaction will not execute, allowing for control.
+
+![Gas fees are negligible compared to collateral.](collateral-gas.png)
```
Comment (johnnymatthews) [inline content/en/storage-provider/how-providing-works/index.md:78]: What's the purpose of this screenshot? It doesn't seem relevant to this page. Maybe we should take it out?

Comment (Annamarie2019) [reply content/en/storage-provider/how-providing-works/index.md:78]: Helps clarify an aspect of costs. Says, "Don't worry, gas fees are negligible."..... But I agree, would be better to have something more relevant, like an example of cost of hardware. Point well taken. We can remove.

File: content/en/storage-provider/how-providing-works/index.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+## Where do providers fit in the Filecoin network
+
+The Filecoin network is a vibrant ecosystem with various roles that support each other, including storage providers, retrieval providers, Filecoin employees, open-source collaborators, notaries, developers, and clients.
+
+Because of blockchain technology, storage clients don’t need a large, centralized intermediary to guarantee the security and privacy of their data. They can trust anyone who wins a Filecoin deal to store their data privately and securely. In fact, the security is greater, because the data is not subject to a single point of failure.
+
+Unlike centralized systems with a large, captive audience from whom they profit, the decentralized Filecoin ecosystem bootstraps and verifies itself so that everyone benefits from the increased value. The main role of Filecoin employees is to facilitate a strong community.
+
+![Thirty-eight hundred storage providers October 2021](ecosystem-data.png)
```
[the remark this answers is not in the document]
Comment (Annamarie2019) [reply content/en/storage-provider/how-providing-works/index.md:197]: I wanted to have an image on the costs and an image on the rewards. The closest I could get was how promising the growing community is. Again, point well taken. We can remove. (I'm concerned that we have so much text and don't help the reader enough by breaking it up. But I'll try harder to find things that are super relevant.)

**Comment by Annamarie2019** [comment]
@johnnymatthews I removed the images. So good to merge.

**Comment by Annamarie2019** [comment]
@johnnymatthews Please merge to fix typos, including a : in a heading. Thanks.

**Review by johnnymatthews — APPROVED** [review]
